src/config/locales/en/error.js

```diff
@@ -1095,5 +1095,11 @@ export const error = {
   validMinimumCitizenships: {
     title: 'There is a problem with your entry',
     message: 'Please provide a minimum of two countries.'
+  },
+  alphanumeric: {
+    pattern: {
+      title: 'There is a problem with your entry',
+      message: 'Only use letters and numbers'
+    }
```

Comment:
Since these fields set a maxlength and are required, I think we'll need to add those validation messages in here under the `alphanumeric` key as well since that prefix is going to get applied to all of the messages.

Reply:
Ah thanks for that!